client/client.go

```diff
@@ -136,6 +146,9 @@ func NewClient(cfg *config.Config) (*Client, error) {
 
 	// Start the client!
 	go c.run()
+
+	// Start the consul client
+	go c.consulClient.SyncWithConsul()
```

Comment:
In this case it doesn't matter, as consul sync will run asynchronously and the client need not wait.